### PR TITLE
Blog: v0.8 release

### PR DESCRIPTION
**Pull request by bleonard**
## Change description

Adds blog post and video about new release.

### Development

- [x] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [ ] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: pages/blog/v08-release.mdx [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: "Grouparoo v0.8 release"
+date: "2022-01-31"
+author: "Brian Leonard"
+pullQuote: "Grouparoo's v0.8 beings a new Model UI and other improvements."
+image: "v08-release/video.jpg"
+tags: [product, engineering]
+---
+
+The v0.8 release is our first major iteration on the UI for creating your data pipeline. In the v0.7 release, we added Models, which allowed data engineers to sync multiple data schemas to Destinations. This release summarizes those Models better in the UI, giving you a clearer overview of the configuration.
```
Comment (evantahler) [inline pages/blog/v08-release.mdx]:
```suggestion
The v0.8 release is our first major iteration on the user interface for creating your data pipeline. In the v0.7 release, we added Models, which allowed data engineers to sync multiple data schemas to Destinations. This release summarizes those Models better in the UI, giving you a clearer overview of the configuration, making it quicker and easier to sync your data.
```

File: pages/blog/v08-release.mdx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+Here are the key features of the release.
+
+- Model-centric UI to see on one page where the data comes from, what it looks like, and where it is going
+- Better filtering for Schedules and Properties
+- Upgrade to Google Sheets Source to act more like SQL Sources
```
Comment (evantahler) [inline pages/blog/v08-release.mdx]:
```suggestion
- Upgraded to Google Sheets Source to act more like SQL Sources
```

File: pages/blog/v08-release.mdx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+- New Destinations: Klaviyo, Vero, Salesforce Accounts and Contacts, Airtable
+
+## Model UI
```
Comment (evantahler) [inline pages/blog/v08-release.mdx]:
```suggestion
## Grouparoo User Interface
```

File: pages/blog/v08-release.mdx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+The main update is in the UI for working with Models. We have moved the navigation to have the Models in it and each Model now has an overview with:
+
+- Model data that includes Sources and Groups
+- Sample record to work with and get a quick sense of the Model schema
+- Destinations where the Model is being sent
+
```
Comment (evantahler) [inline pages/blog/v08-release.mdx:26]: Maybe add a screenshot below the video too?

File: pages/blog/v08-release.mdx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+This outline also follows the flow of data. We hope that, by presenting it this way, you get a better feeling for the pipeline configuration.
+
+As a piece of this, we have also upgraded the sample Record portion of the Model overview. In the open source Config UI, we now automatically add sample Records so it is easier to visualize your schema.
+
+## Filtering Schedules and Properties
+
+Previously, Schedules and Properties could not be filtered by `NULL` values. We have updated the UI and normalized the JSON configuration between these and Group-building.
+
+Now, you can add those `NULL`-ish filters as you like.
```
Comment (evantahler) [inline pages/blog/v08-release.mdx]:
```suggestion
Now, you can add those `NULL`-ish (and non-`NULL`-ish) filters as you like.
```

**Review by evantahler — APPROVED** [review]
Some minor word suggestions, but 👍!

File: pages/blog/v08-release.mdx [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+pullQuote: "Grouparoo's v0.8 beings a new Model UI and other improvements."
+image: "v08-release/video.jpg"
+tags: [product, engineering]
+---
+
+The v0.8 release is our first major iteration on the UI for creating your data pipeline. In the v0.7 release, we added Models, which allowed data engineers to sync multiple data schemas to Destinations. This release summarizes those Models better in the UI, giving you a clearer overview of the configuration.
+
+Here are the key features of the release.
+
+- Model-centric UI to see on one page where the data comes from, what it looks like, and where it is going
```
Comment (andyjih) [inline pages/blog/v08-release.mdx]: Not major: I wonder if "Model-centric UI" is too much of an internal term or phrasing.
"Model-based UI"
"Model-focused UI"

Wondering if maybe I'm too close to it though.

File: pages/blog/v08-release.mdx [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+    src="https://www.youtube.com/embed/Pe99Py7LZO8?rel=0"
+    frameBorder="0"
+    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+    allowFullScreen
+  ></iframe>
+</p>
+
+This outline also follows the flow of data. We hope that, by presenting it this way, you get a better feeling for the pipeline configuration.
+
+As a piece of this, we have also upgraded the sample Record portion of the Model overview. In the open source Config UI, we now automatically add sample Records so it is easier to visualize your schema.
```
Comment (andyjih) [inline pages/blog/v08-release.mdx]:
```suggestion
As a piece of this update, we have also upgraded the sample Record portion of the Model overview. In the open source Config UI, we now automatically add sample Records so it is easier to visualize your schema.
```

File: pages/blog/v08-release.mdx [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+  src="v08-release/filter.png"
+  width={533}
+  height={200}
+/>
+
+See the [upgrade notes](https://www.grouparoo.com/docs/support/upgrading-grouparoo/v07-v08) for information on how to update your configurations.
+
+## Plugins
+
+[Google Sheets](https://www.google.com/sheets/about/) now acts more like SQL Sources. For example, this allows it to work better in fetching single rows from the sheet, keeping your data more consistent.
```
Comment (andyjih) [inline pages/blog/v08-release.mdx]:
```suggestion
[Google Sheets](https://www.google.com/sheets/about/) now acts more like SQL Sources like MySQL and PostgresQL. For example, this update allows the plugin to work better in fetching single rows from the sheet, keeping your data more consistent.
```

**Review by andyjih — CHANGES_REQUESTED** [review]
Minor comments, but looking good!

**Review by andyjih — APPROVED** [review]

**Pull request merged**